sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1124,7 +1128,7 @@ private synchronized void handleApiCallSuccess(Object releaseCallId, String rawR
 
                 /* Check version code is equals or higher and hash is different. */
                 AppCenterLog.debug(LOG_TAG, "Check if latest release is more recent.");
-                if (isMoreRecent(releaseDetails) && canUpdateNow(releaseDetails)) {
+                if (isMoreRecent(mPackageInfo, releaseDetails) && canUpdateNow(releaseDetails)) {
```

Comment:
why passing a member field to a member method parameter?

Reply:
it's static utils method now. I moved utils method with checks and url building to utils since they do not depend on syncs or many fields. They are good fit for utils class:

https://github.com/microsoft/appcenter-sdk-android/blob/69306570845e0a25db23017bb498178d673a50a0/sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeUtils.java#L40-L42